packages/dynamite/dynamite/lib/src/openapi_builder.dart

```diff
@@ -78,7 +78,9 @@ class OpenAPIBuilder implements Builder {
             Directive.import('dart:typed_data'),
             Directive.import('package:built_collection/built_collection.dart'),
             Directive.import('package:built_value/json_object.dart'),
-            Directive.import('package:collection/collection.dart'),
+            // The allocator will always start at _i1.
+            // It is safe to use 0.
+            Directive.import('package:collection/collection.dart', as: '_i0'),
```

Comment:
So this is manually imported and only used for extensions, right?

Reply:
yes.
Thinking more about it I we should probably switch it to: `_i0.IterableExtension(authentications)?.firstWhereOrNull`
This removes the special case and will only add the import if used (helping us with #1244)